pinot-common/src/main/java/org/apache/pinot/common/function/scalar/StringFunctions.java

```diff
@@ -119,6 +131,17 @@ public static String concat(String input1, String input2, String seperator) {
     return result;
   }
 
+  /**
+   * Join two input string with no seperator in between
+   * @param input1
+   * @param input2
+   * @return The two input strings joined
+   */
+  @ScalarFunction
+  public static String concat(String input1, String input2) {
+    return concat(input1, input2, "");
+  }
```

Comment:
we can support similar to casewhen --> e.g. have arg support up to say 5 strings and then it will throw)

Reply:
we can't because it has the same type signature as the existing concat, which has incorrect behavior (e.g. `concat(String, String, String)` already treats the third string as the delimiter)

Reply:
Ah.